flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/subtask/job-checkpoints-subtask.component.html

```diff
@@ -181,7 +186,19 @@
           <td>{{ subTask['unaligned_checkpoint'] }}</td>
         </ng-container>
         <ng-container *ngIf="subTask['status'] === 'pending_or_failed'">
-          <td colspan="8">n/a</td>
+          <td colspan="10">n/a</td>
+          <td>
+            <a
+              *ngIf="mapOfSubtask.get(subTask['index'])"
+              [routerLink]="[
+                '/task-manager',
+                mapOfSubtask.get(subTask['index'])['taskmanager-id'],
+                'logs'
```

Comment:
I noticed the link has changed to `logs`, would you please share the new picture with this change?

Reply:
Yes, I have updated the picture above.